python/lsst/pipe/tasks/multiBand.py

```diff
@@ -830,10 +830,12 @@ def getSkySourceFootprints(self, mergedList, skyInfo, growDetectedFootprints=0):
 
             x = int(numpy.random.uniform(xmin, xmax))
             y = int(numpy.random.uniform(ymin, ymax))
-            foot = afwDetect.Footprint(afwGeom.PointI(x, y), skySourceRadius, patchBBox)
+            spans = afwGeom.SpanSet.fromShape(int(skySourceRadius),
+                                              offset=(x, y))
+            foot = afwDetect.Footprint(spans, patchBBox)
             foot.setPeakSchema(self.merged.getPeakSchema())
 
-            if not foot.overlapsMask(mask):
+            if not foot.spans.overlaps(afwGeom.SpanSet.fromMask(mask)):
```

Comment:
I don't know that I like this change. I think that ``SpanSet.overlaps`` should be overloaded to accept a mask, much like ``SpanSet.intersect``, to prevent the user from having to unnecessarily cast a ``Mask`` to a ``SpanSet``.